src/chrome/chromeConnection.ts

```diff
@@ -155,7 +155,7 @@ export class ChromeConnection {
                 if (Array.isArray(responseArray)) {
                     // Filter out extension targets and other things
                     // Non-chrome scenarios don't always specify a type, so filter to include ones without a type at all
-                    let pages = responseArray.filter(target => target && (!target.type || target.type === 'page'));
+                    let pages = responseArray;//.filter(target => target && (!target.type || target.type === 'page'));
```

Comment:
Yes I agree. But keep in mind that even with pages, attach to a wrong tab happens all the time so will do with extensions... That is something on chrome debugger defined as "by design" that you can't avoid... People using other tools suffer with the same today even developing pages... Many other tools ask for the user to close all chrome instances before start the debug... You have the "url" setting on launch.json which should avoid to close chrome if already open... Lets just emphasize on the docs that url must be used to find the tab...